provisioning/device/src/tpm_registration.ts

```diff
@@ -1,20 +1,23 @@
 import { EventEmitter } from 'events';
 import * as machina from 'machina';
 import * as dbg from 'debug';
-const debug = dbg('azure-iot-provisioning-device:TpmRegistration');
+const debug = dbg('azure-device-provisioning:TPM');
```

Comment:
>TPM [](start = 45, length = 3)

Nope. the convention is to use the class name here. I'd rather see this reverted.